appservice/src/main/java/nu/yona/server/BruteForceAttemptService.java

```diff
@@ -1,136 +0,0 @@
-/*******************************************************************************
```

Comment:
OK, it is not necessary at device requests because these use a user ID and should be short-living.